server/server_test.go

```diff
@@ -90,6 +91,34 @@ func TestGetKeysEndpoint(t *testing.T) {
 	}
 }
 
+// RotateKey supports only the timestamp and snapshot key endpoints
+func TestRotateKeyEndpoint(t *testing.T) {
+	ctx := context.WithValue(
+		context.Background(), "metaStore", storage.NewMemStorage())
+	ctx = context.WithValue(ctx, "keyAlgorithm", data.ED25519Key)
+
+	handler := RootHandler(nil, ctx, signed.NewEd25519())
+	ts := httptest.NewServer(handler)
+	defer ts.Close()
+
+	rolesToStatus := map[string]int{
+		data.CanonicalTimestampRole: 429, // not implemented yet
+		data.CanonicalSnapshotRole:  429, // not implemented yet
```

Comment:
(Even though it may not be used here after the snapshot rotation is implemented, but it could be nice)